src/lib/services/addon-service.test.ts

```diff
@@ -627,7 +628,7 @@ test('should reject addon config with missing required parameter when creating',
 
     await expect(async () =>
         addonService.createAddon(config, 'me@mail.com'),
-    ).rejects.toThrow(ValidationError);
+    ).rejects.toThrow(BadDataError);
```

Comment:
We are handling ValidationError by returning 400, it feels like that is more verbose and explicit.